ht-12/index.js

```diff
@@ -0,0 +1,77 @@
+function carousel () {
+  const nextButtons = Array.from(document.querySelectorAll(`[data-slide="next"]`));
+  const prevButtons = Array.from(document.querySelectorAll(`[data-slide="prev"]`));
+  let carouselWrapper;
+  let carouselItems;
+  nextButtons.forEach(nextButton => {
+    nextButton.addEventListener('click', () => {
+      carouselWrapper = document.querySelector(`${nextButton.dataset.target}`);
+      carouselItems = Array.from(carouselWrapper.querySelectorAll('.carousel-item'));
+      slideLeft();
+    });
+  });
+  prevButtons.forEach(prevButton => {
+    prevButton.addEventListener('click', () => {
+      carouselWrapper = document.querySelector(`${prevButton.dataset.target}`);
+      carouselItems = Array.from(carouselWrapper.querySelectorAll('.carousel-item'));
+      slideRight();
+    });
+  });
+  function slideLeft() {
```

Comment:
the same. let's unify the slideLeft/slideRight function. that should be one function that accept direction as a parameter